templates/municipal_chief/municipal/volunteer/detail.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends 'municipal_chief/_layout.html.twig' %}
+
+{% block municipal_chief_content %}
+    <a href="{{ path('app_municipal_chief_municipal_volunteer_request') }}" class="back-to-list icon--with-text b__nudge--top-15 b__nudge--bottom">
+        {{ include('components/caret--left.html.twig') }}
+        Retour à la liste des candidats volontaires
+    </a>
+
+    <hr />
+
+    {{ include('referent/municipal/volunteer/_detail.html.twig') }}
```

Comment:
could you move this template in `/templates/municipal/` directory as it is used by two spaces: municipal_chief and referent
and we can rename it in `_volunteer_detail.html.twig`
same remark for the running_mate_detail template.

Reply:
As we disused i placed those partials in `templates/application_request/municipal/`